mvpa2/measures/rsa.py

```diff
@@ -199,12 +202,15 @@ class PDistTargetSimilarity(Measure):
           If True, return only the correlation coefficient (rho), otherwise
           return rho and probability, p.""")
 
-    def __init__(self, target_dsm, **kwargs):
+    def __init__(self, target_dsm, control_dsms = None, **kwargs):
```

Comment:
very minor nit-picking:  there should be no spaces around = within keyword definitions of 'def'.
Also all the optional parameters to this, as you can see above this line, specified as "Parameters".